src/index.css

```diff
@@ -1,13 +1,27 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family:
```

Comment:
- [ ] OPTIONAL: Your design works, but I would recommend you improve it to be as faithful as possible to the given design.  Some sections could be improved, like the "Add Motorcycle' section.  You could add some shadows, some rounding to the forms, work a little on the fonts styling.

Reply:
hey @Meltrust. since this is optional i would love to take my time to address this. thank you. and regarding the deployment we are having some difficulty deploying from the main branch for now. so there is a seperate branch we decided to do this from. thanks

Reply:
If you are going to make other changes to a separate branch, please remember to link to this PR so the next reviewer can see this review, or else he/she will get confused.  Also, you could do whatever you want in another branch and then merge with this one.